perf/perf_cpu_hotplug.py

```diff
@@ -135,6 +137,28 @@ def test_cpumask_cpu_off(self):
             self.cpu_off.append(disable_cpu)
         self._check_cpumask()
 
+    def test_cpumask_cpu_off_random(self):
+        for i in range(0, len(self.online_cpus)-1):
+            cpuno = random.choice(self.online_cpus)
+            self._cpu_on_off(cpuno)
+            self.cpu_off.append(cpuno)
+            new_hv_24X7_cpu = self._get_cpumask("hv_24x7")
+            if new_hv_24X7_cpu not in self.online_cpus:
+                self.fail("hv_24x7 points to an offline cpu")
```

Comment:
@akanksha216 can you add more print statements like current cpumask for 24/7 and online cpu list after offlining.
from job log file its not clear if its working as expected or not.
```
[stdlog] 2024-04-23 15:10:22,242 avocado.test perf_cpu_hotplug L0102 INFO | Online CPU list: [0, 1, 2, 3, 4, 5, 6, 7]
[stdlog] 2024-04-23 15:10:22,488 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 3
[stdlog] 2024-04-23 15:10:22,608 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 5
[stdlog] 2024-04-23 15:10:22,608 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 5
[stdlog] 2024-04-23 15:10:22,726 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 0
[stdlog] 2024-04-23 15:10:22,726 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 3
[stdlog] 2024-04-23 15:10:22,727 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 5
[stdlog] 2024-04-23 15:10:22,828 avocado.test perf_cpu_hotplug L0115 INFO | Offline CPU: 2
[stdlog] 2024-04-23 15:10:22,829 avocado.utils.process process          L0655 INFO | Running 'dmesg'
```